### PR TITLE
React Setup

### LINKED ISSUE #1: React Setup
Simple task to setup a react environment.

### PR DESCRIPTION
**Pull request by KacperWojdak**
Done doing simple React Setup.

### CODE REVIEW

File: kebab_city_tour/public/logo192.png
Comment (PiotrFedak) [inline kebab_city_tour/public/logo192.png:1]: I think this should be deleted, we don't need that

File: kebab_city_tour/public/logo512.png
Comment (PiotrFedak) [inline kebab_city_tour/public/logo512.png:1]: Also this

File: kebab_city_tour/public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (PiotrFedak) [inline kebab_city_tour/public/index.html]: Change Title to Kebab something instead of React App

File: kebab_city_tour/src/logo.svg
Comment (PiotrFedak) [inline kebab_city_tour/src/logo.svg:1]: Also deleate this file

**Review by PiotrFedak — CHANGES_REQUESTED** [review]

**Review by AndrzejBudzyn — CHANGES_REQUESTED** [review]
Change README

**Review by PiotrFedak — APPROVED** [review]
Okey

**Review by AndrzejBudzyn — APPROVED** [review]
know the lord's grace

**Pull request merged**